sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/splittabledofn/Backlog.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.transforms.splittabledofn;
+
+import com.google.auto.value.AutoValue;
+import java.util.Arrays;
+import javax.annotation.Nullable;
+
+/**
+ * A representation for the amount of known work represented as a backlog.
```

Comment:
Can you specify here the scope of a backlog? i.e., does the Backlog specify the "known work" of..

1. an element+restriction pair?
1. a single input element
1. a bundle application
1. \<something else\>

And does the Backlog explicitly refer to known work that hasn't yet been completed, or hasn't yet been delivered from the runner, or the total work including processed and unprocessed ..?

From my understanding, I believe a Backlog represents the unclaimed portion of an element restriction.

Reply:
Got it; thanks for the clarification.